### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by ckhatton** [geddy/geddy#727]
Who is in charge with tagging and editing the changelog?

If anyone is, are we going by the [vMajor.Minor.Hotfix] scheme of version numbering?

e.g.
> v13.0.9

**Comment by mde** [comment]
Yes, we're using the traditional semver scheme. The tagging and release process is done automatically (by me) using a Jake task: https://github.com/geddy/geddy/blob/master/Jakefile#L46 (the `npmPublishTask` is an older name for Jake's `PublishTask` (http://jakejs.com/docs#publish_task). IIRC, the changelog is just manually edited. I'm happy to delegate some of this process, but we have to be very careful to make sure we don't accidentally publish a broken version to NPM.
